.github/workflows/shared_deploy_cloudhub_1_0.yml

```diff
@@ -48,6 +57,21 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      # Remove when switching to the new Exchange - BUSINESS_GROUP_ID_GLOBAL_2 completely for all ENV (DEV, STAGING, PROD)
```

Comment:
This is similar to the assets topic, we now have to have the Mule apps running on 2 different Business groups

- DEV + STAGING => BUSINESS_GROUP_ID_GLOBAL
- PROD => BUSINESS_GROUP_ID_GLOBAL_2

Later, there is a plan to move all `DEV + STAGING` apps to the `BUSINESS_GROUP_ID_GLOBAL_2` 👍